src/aiida_quantumespresso/calculations/bands.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+"""`CalcJob` implementation for the q2r.x code of Quantum ESPRESSO."""
+
+from aiida import orm
+
+from aiida_quantumespresso.calculations.namelists import NamelistsCalculation
+
+
+class BandsCalculation(NamelistsCalculation):
+    """`CalcJob` implementation for the bands.x code of Quantum ESPRESSO.
+
+    bands.x code of the Quantum ESPRESSO distribution, re-orders bands, and computes band-related properties.
+
+    It computes for instance the expectation value of the momentum operator:
+    <Psi(n,k) | i * m * [H, x] | Psi(m,k)>. For more information, refer to http://www.quantum-espresso.org/
+    """
+
+    _MOMENTUM_OPERATOR_NAME = 'momentum_operator.dat'
+    _BANDS_NAME = 'bands.dat'
+
+    _default_namelists = ['BANDS']
+    _blocked_keywords = [
+        ('BANDS', 'outdir', NamelistsCalculation._OUTPUT_SUBFOLDER),  # pylint: disable=protected-access
+        ('BANDS', 'prefix', NamelistsCalculation._PREFIX),  # pylint: disable=protected-access
+        ('BANDS', 'filband', _BANDS_NAME),  # pylint: disable=protected-access
```

Comment:
I think this rule is not used, right?
```suggestion
        ('BANDS', 'filband', _BANDS_NAME),
```

Reply:
Right, thanks!